int64 id
string name
geometry_msgs/Point position
geometry_msgs/Point radius3D
float64 radius 
string motion_model
geometry_msgs/Point state_space_model_position
geometry_msgs/Twist state_space_model_velocity
float64 dt
bool display_solid
bool is_advanced
string type 
float64 max_speed
BoundingBox limits
bool predictable
TrajectoryState[] trajectory

